worlds/seaofthieves/Regions/Regions.py

```diff
@@ -0,0 +1,88 @@
+import copy
+import typing
+
+from BaseClasses import MultiWorld, Region, Entrance
+from worlds.seaofthieves.Locations.Locations import *
+from ..Locations.LocationCollection import LocationDetailsCollection
+from ...generic.Rules import add_rule, exclusion_rules
+from ..Items.Items import Items
+from ..Configurations import SotOptionsDerived
+import typing
+from worlds.seaofthieves.Regions.RegionDetails import RegionDetails, Regions
+from worlds.seaofthieves.Regions.ConnectionDetails import ConnectionDetails
```

Comment:
```suggestion
from .RegionDetails import RegionDetails, Regions
from .ConnectionDetails import ConnectionDetails
```
Use relative imports when importing from within your own world.